.github/workflows/krew.yaml

```
@@ -1,8 +1,9 @@
 name: Push to Krew
 
 on:
-  release:
-    types: [published]
+  push:
+    tags:
+      - '*'
```

We can make this a bit stronger:

```suggestion
      - 'v*.*.*'
```